tests/testthat/test_AutoTuner.R

```diff
@@ -275,7 +273,7 @@ test_that("search space from TuneToken works", {
 
 test_that("AutoTuner get_base_learner method works", {
   skip_if_not_installed("mlr3pipelines", "0.5.3")
-  requireNamespace("mlr3pipelines")
+  `%>>%` = getFromNamespace("%>>%", asNamespace("mlr3pipelines"))
```

Comment:
nit: this make it look like `%>>%` is private, but it's exported:

https://github.com/mlr-org/mlr3pipelines/blob/9d4cced9b7b887db6c80c4fafd861398f727cf29/NAMESPACE#L84

(`asNamespace()` gives all objects in the package, including those not exported in NAMESPACE)

I would just use:

```r
`%>>%` = mlr3pipelines::`%>>%`
```

(same for elsewhere that I see)

Reply:
Yes that's better. Thanks